public_html/administrator/components/com_jed/models/reviews.php

```diff
@@ -32,17 +32,28 @@ public function __construct($config = [])
 		if (empty($config['filter_fields']))
 		{
 			$config['filter_fields'] = [
-				'published', 'reviews.published',
-				'created_by', 'reviews.created_by',
-				'created_on', 'reviews.created_on',
-				'title', 'reviews.title',
-				'score', 'reviews.score',
-				'developer', 'users.username',
-				'reviewer', 'users.id',
-				'extension', 'extensions.extension_id',
-				'title', 'extensions.title',
-				'ipaddress', 'reviews.ipAddress',
-				'id', 'reviews.id',
+				'published',
+				'reviews.published',
+				'created_by',
+				'reviews.created_by',
+				'created_on',
+				'reviews.created_on',
+				'title',
+				'reviews.title',
+				'score',
+				'reviews.score',
+				'developer',
+				'users.username',
+				'reviewer',
+				'users.id',
+				'extension',
+				'extensions.extension_id',
+				'title',
+				'extensions.title',
+				'ipaddress',
+				'reviews.ipAddress',
+				'id',
+				'reviews.id',
```

Comment:
Unfortunately this doesn't work :)

This list is composed of basically two entries:
1. The fields from your search tools
2. The fields from your table headers

So for part 1, we will need:

- published
- reviewer
- extension
- developer
- ipaddress
- flagged

These are just the names of the fields as defined in the filter_reviews.xml

For part 2, we will need:

- reviews.published
- reviews.created_date
- reviews.title
- reviews.overall_score
- users.username
- extensions.title
- extensions.created_by
- reviews.ipAddress
- reviews.flagged
- reviews.id

Remove everything that is in the array now and add these two lists, keeping them as separate lists makes it easier to maintain as you can instantly see which field is for what part of the site.

With this new array, your search tools will remain open and the column headers will sort.